android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -125,54 +121,44 @@ public void setHttpHandler(HttpHandler handler) {
      * Set the geographic position of the center of the map view
      * @param position LngLat of the position to set
      */
-    public void setMapPosition(LngLat position) {
-        setMapPosition(position.longitude, position.latitude);
+    public void setPosition(LngLat position) {
+        nativeSetPosition(position.longitude, position.latitude);
     }
 
     /**
      * Set the geographic position of the center of the map view
-     * @param lng Degrees longitude of the position to set
-     * @param lat Degrees latitude of the position to set
-     */
-    public void setMapPosition(double lng, double lat) {
-        nativeSetPosition(lng, lat);
-    }
-
-    /**
-     * Set the geographic position of the center of the map view
-     * @param lng Degrees longitude of the position to set
-     * @param lat Degrees latitude of the position to set
-     * @param duration Time in seconds to ease to the given position
+     * @param position LngLat of the position to set
+     * @param duration Time in milliseconds to ease to the given position
      */
-    public void setMapPosition(double lng, double lat, float duration) {
-        nativeSetPositionEased(lng, lat, duration, EaseType.QUINT.ordinal());
+    public void setPosition(LngLat position, int duration) {
+        setPosition(position, duration, DEFAULT_EASE_TYPE);
     }
 
     /**
      * Set the geographic position of the center of the map view
-     * @param lng Degrees longitude of the position to set
-     * @param lat Degrees latitude of the position to set
-     * @param duration Time in seconds to ease to the given position
+     * @param position LngLat of the position to set
+     * @param duration Time in milliseconds to ease to the given position
      * @param ease Type of easing to use
      */
-    public void setMapPosition(double lng, double lat, float duration, EaseType ease) {
-        nativeSetPositionEased(lng, lat, duration, ease.ordinal());
+    public void setPosition(LngLat position, int duration, EaseType ease) {
+        float seconds = duration / 1000.f;
```

Comment:
So internally its still in seconds. Just curious if the switch is to have compat with gmaps?